client/src/components/App/app.jsx

```diff
@@ -27,6 +31,32 @@ const Button = styled.button`
 
 `;
 
+
+// const ModalWithTransitionStyles =styled(Modal)`
+// &.modal-transition-enter{
+//   transform: translateY(-100%);
+// }
+// &.modal-transition-enter-active{
+//   transition: transform 900ms ease-in-out ;
+//   transform:translateY(0)
+// }
+
+// &.modal-transition-exit{
+//   transform:translateY(0):
+// }
+// &.modal-transition-exit-active{
+//   transition:transform 900ms ease-in-out ;
+//   transform: translateY(-100%);
+// }
+
+//  &.modal-transition-appear
+
+// `;
```

Comment:
why are these commented out?

Reply:
I was working on adding animation for my modal but I could not make it to work.  For right now I just commented it out and will go back to try to make it work